commit 7eaab98d69d8bdbc46cceaeac64007b214593174
Author: lukas227 <lukas-gh@schwaighofer.name>
Date:   Mon Aug 13 22:35:27 2012 +0200

    fixed legacy persistence handling
    
    * the mount_persistence_media () function must return the mount point of
      the partition to the caller, even if it was already mounted
      (${backing} = ${old_backing})

diff --git a/scripts/live-helpers b/scripts/live-helpers
index b90637a..7d5c675 100644
--- a/scripts/boot/9990-misc-helpers.sh
+++ b/scripts/boot/9990-misc-helpers.sh
@@ -778,6 +778,7 @@ mount_persistence_media ()
 			return 1
 		fi
 	fi
+	echo ${backing}
 	return 0
 }
 
